### DISCUSSION TITLE
Turning off console output for http sink

### DISCUSSION THREAD

**Discussion by rterbush** [vectordotdev/vector#21901] [Q&A]
Is it possible to turn off the response from the http sink? Trying to debug things here but console is flooded by the output from this sink. Attempting to sink to Quickwit if that matters.

**Comment by pront** [comment]
> Is it possible to turn off the response from the http sink? 

No. I usually use a fake HTTP server that accepts everything for such cases before changing to the the real endpoint.

**Comment by jszwedko** [comment]
Can you share the flooding output in question? I'm not sure what logs we are talking about.
